src/Badge/Model/UseCase/CreateDependentsBadge.php

```diff
@@ -59,8 +59,8 @@ public function createDependentsBadge(string $repository, string $format = 'svg'
             $smaxage
         );
 
-        $subject = $badge->getSubject();
-        $order = \substr($subject, -1);
+        $status = trim($badge->getStatus());
+        $order = \substr($status, -1);
```

Comment:
since when did we introduce this bug?

Reply:
Since my previous PR :(
The effect of it was not so negative, it was just caching for less time than expected.